Tests/FluentAssertions.Specs/Primitives/StringAssertionSpecs.cs

```diff
@@ -3243,5 +3243,220 @@ public void When_a_whitespace_string_is_expected_to_not_be_null_or_whitespace_it
         }
 
         #endregion
+
+        #region (Not) Upper
+
+        [Fact]
+        public void Should_succeed_when_asserting_upper_string_to_be_upper()
```

Comment:
Our convention is `When_something_it_should_do_something`. I know we have some older tests that don't follow that convention, but we need to change that,

Reply:
Do you mean `When_asserting_uppercase_string_to_be_upper_should_succeed`, in this case? Just to be clear.

Reply:
Almost ;-)

`When_an_upper_case_string_is_supposed_to_be_in_upper_case_only_it_should_not_throw`